pages/conditional-includes.md

```diff
@@ -0,0 +1,92 @@
+import { Meta } from '../components'
+
+<Meta
+  title="Keeping up with two git identities"
+  description="You can work with your company projects and with open-source projects without mixing two identities."
+/>
+
+# Keeping up with two git identities
+
+_November 13, 2018_
+
+If you are working in the frontend, then you most likely use a lot of open-source projects and perhaps your company even open sourced several internal projects. Also you most likely fix issues you have with 3rd party projects you depend on from time to time.
```

Comment:
* "most likely" repeats twice, maybe you could use "chances are" or "probably"
* "3rd party projects you depend on" -> "3rd party dependencies"

Reply:
fixed